src/app/shared/register-github-app/register-github-app.component.html

```diff
@@ -0,0 +1,38 @@
+<div>
+  <p>
+    Install the Dockstore GitHub app on your repositories. This gives GitHub permission to notify Dockstore when pushes are made to GitHub.
+    For more details, see our
+    <a href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/github-apps/github-apps.html" target="_blank" rel="noopener noreferrer">
+      documentation <mat-icon>open_in_new</mat-icon> </a
+    >.
+  </p>
+  <p>Perform the following steps in order:</p>
+  <ol>
+    <li>Click the button below to install the Dockstore GitHub app on your GitHub organizations or specific repositories.</li>
+    <li>
+      Push a valid .dockstore.yml to the root of a GitHub repo that has the Dockstore GitHub app installed. The .dockstore.yml tells
+      Dockstore where the {{ entryType }}'s files are located.
+    </li>
+    <li>
+      It takes about 2 minutes after a push to GitHub for Dockstore to get notified and process the update. After those 2 minutes, refresh
+      your dashboard, and you should see the {{ entryType }}.
```

Comment:
"refresh your dashboard", is this the My <s>Dashboard</s> Dockstore page? Should we instead suggest the My <entry> page?

Reply:
The case I'm worried about is that you're already on My Workflows, which isn't unreasonable -- it's a good place from where to create a workflow. And clicking on My Workflows if you're already on it won't refresh.